spirv_cross/src/glsl.rs

```diff
@@ -91,6 +93,7 @@ impl Default for CompilerOptions {
     fn default() -> CompilerOptions {
         CompilerOptions {
             version: Version::V4_50,
+            no_420_pack_extension: false,
```

Comment:
Could we use the same form as the C++ library and default to `true`? So something like `enable_420_pack_extension` instead

Reply:
sure thing, i commited.
note: i didn't do any tests locally for this change